googletest/include/gtest/internal/gtest-port.h

```diff
@@ -874,6 +874,23 @@ using ::std::tuple_size;
 # define GTEST_ATTRIBUTE_UNUSED_
 #endif
 
+// Use this annotation before a function that takes a printf format string.
```

Comment:
I'd rather not introduce this complex definition, and we will just eyeball the printf formats, which are all internal.